samples/videoDecodeRGB/CMakeLists.txt

```diff
@@ -56,9 +56,18 @@ else()
   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -O3 -DNDEBUG -fPIC")
 endif()
 
-# set amdgpu targets with vcn support
+# Set supported GPU Targets
 set(DEFAULT_AMDGPU_TARGETS "gfx908;gfx90a;gfx940;gfx941;gfx942;gfx1030;gfx1031;gfx1032;gfx1100;gfx1101;gfx1102")
-set(AMDGPU_TARGETS "${DEFAULT_AMDGPU_TARGETS}" CACHE STRING "List of specific machine types for library to target")
+
+# Set AMDGPU_TARGETS
+if(DEFINED ENV{AMDGPU_TARGETS})
+  set(AMDGPU_TARGETS $ENV{AMDGPU_TARGETS} CACHE STRING "List of specific machine types for library to target")
+elseif(AMDGPU_TARGETS)
+  message("-- ${White}${PROJECT_NAME} -- AMDGPU_TARGETS set with -D option${ColourReset}")
```

Comment:
probably we set or display should display possible GPU targets for user. I am not sure if user awares of gfx ids for building rocDecode.